app/celery_utils.py

```diff
@@ -0,0 +1,27 @@
+import json
```

Comment:
instead of `celery_utils` maybe just `tasks` or `property_operators`

I said they can be organized like utility functions in modules that are named in relation to what they do
and not necesarily adding utils to the name itself